app/javascript/mastodon/components/display_name.tsx

```diff
@@ -56,13 +55,13 @@ export default class DisplayName extends React.PureComponent {
         account = this.props.account;
       }
 
-      let acct = account.get('acct');
+      let acct = account?.get('acct');
```

Comment:
Is the `?` needed here? This means that `acct` might be `undefined`, and the next line (`acct.indexOf`) will fail in this case.

It would be best to properly throw an exception (or skip the code) if `!account` before this line, so Typescript properly knows that `account` is always defined after.